index.html

```diff
@@ -0,0 +1,166 @@
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Iqbal Day Poetry Festival</title>
+  <meta name="description" content="Portal Capstone">
+  <meta name="author" content="Humaira Zaman">
+  <meta name="keywords" content="html tutorial template">
+
+  <link rel="stylesheet" href="styles.css?v=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="http://fonts.cdnfonts.com/css/cocogoose">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;1,300;1,400;1,700&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="mobile" id="hamburger"><b>&#9776;</b></div>
+      <div class="bothnav">
+        <div class="social desktop">
+          <ul>
+            <li><a href="#" class="fa fa-facebook"></a></li>
+            <li><a href="#" class="fa fa-twitter"></a></li>
+            <li><select name="language" id="lang">
+                  <option value="eng" selected>English</option>
+                  <option value="urdu">Urdu</option>
+                  <option value="farsi">Farsi</option>
+                </select>
+              </li>
+            <li class="loginBtn">Login / Sign up</li>
+          </ul>
+        </div>
+        <div class="navbar hamburgerNav">
+          <a href="#homepage"><img class="logo desktop" src="./img/logo.png" alt="Logo"></a>
+          <ul>
+            <li><a class="navLink aboutpage" href="#aboutpage">About</a></li>
+            <li><a class="" href="#">Program</a></li>
+            <li><a class="navLink featuredpage" href="#featuredpage">Join</a></li>
+            <li><a class="" href="#">Sponsor</a></li>
+            <li><a class="" href="#">News</a></li>
+            <li class="mobile empty">_empty_</li>
+            <li class="mobile loginBtn">Log in/Sign up</li>
+            <li class="mobile empty">_empty_</li>
+            <li><a id="homelink" href="#homepage">Iqbal Day</a></li>
+            <li class="mobile empty">_empty_</li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </header>
+
+  <main>
+    <section class="home" id="homepage">
+      <h5>"Welcome, Poets All!"</h5>
+      <h1>ALLAMA IQBAL DAY<br>POETRY FESTIVAL<br>2021</h1>
+      <p>A joyful day celeberating a great man with the dream of Pakistan
+        and an independent state for the Muslims of the subcontinent.
+        His golden words had the power to encourage the people to fight
+        for their rights and take a step forward for freedom.</p>
+      <h3>2021.11.09 (TUE)</h3>
+      <h4>@ Allama Iqbal University, Lahore</h4>
+    </section>
+
+    <section class="about" id="aboutpage">
```

Comment:
- [ ] Please make sure the about page contains a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo👌 and a bottom section with a centred column that includes a section header, a divider, a line of text, and two images with text centred over them😉.
- design example
![screenshot (19)](https://user-images.githubusercontent.com/44624138/142292239-3fa3ba6d-358c-482b-b8d5-0fe88f0e4fb8.png)

Reply:
Kindly note that on mobile the bottom section images are organized in a vertical column✔ and on desktop, the bottom section images are organized in a horizontal row🙄. 
- example
![screenshot (20)](https://user-images.githubusercontent.com/44624138/142292750-1558f9bc-535c-49e7-bf64-70bb83a6f434.png) ![screenshot (21)](https://user-images.githubusercontent.com/44624138/142292773-2eb2db2d-6255-475e-ab1b-92f35d9df931.png)